platform-server-kernel/src/main/java/uno/cod/platform/server/core/filter/AccessTokenAuthenticationFilter.java

```diff
@@ -0,0 +1,50 @@
+package uno.cod.platform.server.core.filter;
+
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.filter.OncePerRequestFilter;
+import uno.cod.platform.server.core.service.AccessTokenService;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AccessTokenAuthenticationFilter extends OncePerRequestFilter {
+    private AccessTokenService accessTokenService;
+
+    public AccessTokenAuthenticationFilter(AccessTokenService accessTokenService) {
+        this.accessTokenService = accessTokenService;
+    }
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws ServletException, IOException {
+        final boolean debug = logger.isDebugEnabled();
+
+        String header = request.getHeader("Authorization");
+
+        if (header == null || !header.startsWith("Token ")) {
```

Comment:
Note that you are using an unregistered authentication scheme here. Maybe add this as a comment for future reference. Registered authentication schemes can be found at http://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml